trojsten/submit/views.py

```diff
@@ -499,12 +505,96 @@ def task_submit_post(request, task_id, submit_type):
             return redirect(request.POST["redirect_to"])
         else:
             return redirect(reverse("task_submit_page", kwargs={"task_id": int(task_id)}))
-
+    elif submit_type == constants.SUBMIT_TYPE_TEXT:
+        return task_submit_post_susi(request, task_id, submit_type)
```

Comment:
ja by som tomu textovemu submit view spravil zvlast URL-ku a z templatu sa odkazoval priamo na nu.., takto ked to chce clovek precitat ze co sa stane ked submitnes text, tak sa najskor musi prescrollovat cez 120 riadkov irelevantnych if-iek (kedze nic sa z nich nepouzije)..